drivers/sdhc/Kconfig.rcar

```diff
@@ -14,7 +14,6 @@ config RCAR_MMC_DMA_SUPPORT
 	bool "Internal DMA support for Renesas Rcar MMC driver"
 	select CACHE_MANAGEMENT
 	select DCACHE
-	depends on !SHIELD_XENVM_DOM0
```

Comment:
Please, add more description for this commit ("We needed to turn of DMA due to cache..., now it has a workaround and we can...") after some time we will probably forget about a problem, that caused this changes.